src/components/WwwFrame/TheHeader.vue

```diff
@@ -647,6 +674,8 @@ export default {
 			basketTotal: 0,
 			teams: null,
 			teamsMenuEnabled: false,
+			newAddToBasketExpVersion: 'a',
```

Comment:
Would it be clearer to store whether the experiment is enabled, like we often do, or will we have a version other than a + b?